d3a_interface/data_classes.py

```diff
@@ -131,6 +132,24 @@ def serializable_dict(self) -> Dict:
                 "seller_id": self.seller_id,
                 }
 
+    @staticmethod
+    def from_dict(offer: Dict) -> "Offer":
+        """Deserialize an offer dict."""
+
+        return Offer(
+            **{"id": offer["id"],
```

Comment:
Nit: Why not `Offer(id=offer["id"], ...)` ?

Reply:
Right, adapted